config/manifests/bases/update-service-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,102 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |
+      [
+         {
+           "apiVersion": "updateservice.operator.openshift.io/v1",
+           "kind": "UpdateService",
+           "metadata": {
+             "name": "update-service-instance"
+           },
+           "spec": {
+             "replicas": 1,
+             "releases": "openshift-release-dev/ocp-release",
+             "graphDataImage": "quay.io/cincinnati/demo-graph-data:for-testing-only"
+           }
+         }
+       ]
+    capabilities: Basic Install
+    description: Creates and maintains an OpenShift Container Platform Update Service instance
```

Comment:
nit: `s/instance/operator./`?  Do we want to distinguish between the CSV installing the operator and subsequent UpdateService CRs as "instances"?

Reply:
I believe this is describing what the operator installs. I simply changed `Cincinnati` to `OpenShift Container Platform Update Service`. We could change `instance` to `operand`?

Reply:
but the operator doesn't install any operands just landing the CSV.  You need to add a subsequent UpdateService CR to get any operand deployments.  I thought this property was supposed to describe the operator (e.g., maybe there are other operators who have no operand pods, they just manage a CRD and any subsequent CRs directly).  Unfortunately, the CSV declaration has [no docs for this property][1], so if we want to keep `description` about the operands for now, I'm fine with the text as it stands, or with `instance` changed to `operand(s)`.

[1]: https://github.com/operator-framework/api/blob/b55a341f6560db4adec39d69aab1ff3092ea202a/pkg/operators/v1alpha1/clusterserviceversion_types.go#L278

Reply:
I left it "as is" with only the operator name change. As originally written it is describing the operator.